README.md

```diff
@@ -88,10 +88,12 @@ Please read [CONTRIBUTING](./CONTRIBUTING.md) if you are new here or not familia
 2. cd hypertrons-crx
 
 3. yarn install
+   
+4. yarn run prepare
```

Comment:
Hey @wxharry, maybe we can stress that this command will **only be run once** after cloning the repo.